frontend/src/libs/components/star-rating/star-rating.tsx

```diff
@@ -26,13 +32,13 @@ const StarRating: React.FC<Properties> = ({ rating }: Properties) => {
 
   const createStars = (): JSX.Element[] => {
     const stars: JSX.Element[] = [];
-    for (let index = 1; index <= 5; index++) {
+    for (let index = MIN_STARS; index <= MAX_STARS; index++) {
       if (fullFieldStars >= index) {
-        stars.push(createStarItem(100, index));
+        stars.push(createStarItem(FULL, index));
       } else if (allStarts >= index) {
-        stars.push(createStarItem((rating - fullFieldStars) * 100, index));
+        stars.push(createStarItem((rating - fullFieldStars) * FULL, index));
       } else {
-        stars.push(createStarItem(0, index));
+        stars.push(createStarItem(EMPTY, index));
       }
     }
```

Comment:
is it actually used somewhere? if not, delete it please, because we are not having star rating now